### ISSUE TITLE
Rename of column "row_id" -> "row_ids"

### ISSUE THREAD

**Issue by mllg** [DoubleML/doubleml-for-r#64]
The next mlr3 version will include a refactoring which is breaking your package.
The column "row_id" of `as.data.table.Prediction()` will be renamed to "row_ids" (c.f. https://github.com/mlr-org/mlr3/issues/547). 
It would be great if you could update your package accordingly and implement a workaround in the fashion of the following lines to ease the transition:

```r
tab = as.data.table(prediction)
data.table::setnames(tab, old = "row_id", new = "row_ids", skip_absent = TRUE) # rename col for mlr3 <= 0.10.0
```

Thanks and let us know if you are missing some getters or converters.

**Comment by MalteKurz** [comment]
* Thanks for the heads up. We are currently doing a larger refactoring and will then also pick up the reported change in `mlr3`.
* Is there already a planned release date for `0.11.0`?

**Comment by mllg** [comment]
>     * Is there already a planned release date for `0.11.0`?

The sooner, the better. R core changed some stuff in R-devel and is not pressuring me to upload a fixed version. I can also roll back this particular change for the next update, depending on how long your refactoring will take.

**Comment by MalteKurz** [comment]
> > ```
> > * Is there already a planned release date for `0.11.0`?
> > ```
> 
> The sooner, the better. R core changed some stuff in R-devel and is not pressuring me to upload a fixed version. I can also roll back this particular change for the next update, depending on how long your refactoring will take.

Understood, thanks for the explanation. Getting it done this week will be challenging, but we are optimistic to get the changes done and released next week.
So I don't see a reason for a roll-back and you can proceed with your release as planned more or less independent of us.

**Comment by mllg** [comment]
Great, thanks!

**Comment by MalteKurz** [comment]
@mllg I saw the mail from CRAN. We are almost done and try to release a compatible version to CRAN today.

**Comment by MalteKurz** [comment]
Done and released to CRAN in version `0.2.0`.

**Comment by mllg** [comment]
Thanks! And sorry for breaking your package on short notice.
